src/Warnings.hpp

```diff
@@ -110,6 +110,13 @@ THE SOFTWARE.
     #define ABL_PRAGMA_CLANG_DIAGNOSTIC_IGNORED_ZERO_AS_NULL_POINTER_CONSTANT
   #endif
 
+  #if __has_warning("-Wreserved-identifier")
+    #define ABL_PRAGMA_CLANG_DIAGNOSTIC_IGNORED_RESERVED_IDENTIFIER \
```

Comment:
Please define that macro between `ABL_PRAGMA_CLANG_DIAGNOSTIC_IGNORED_RESERVED_ID_MACRO` and `ABL_PRAGMA_CLANG_DIAGNOSTIC_IGNORED_SUGGEST_DESTRUCTOR_OVERRIDE ` to preserve sorting.